lib/standard/bytes.nit

```diff
@@ -0,0 +1,129 @@
+# Introduces classes and methods to treat byte streams and arrays as such
```

Comment:
There is some redundancy in the doc, all modules intro classes and methods... I would recommend the much simpler: `Services for byte streams and arrays`